src/Stryker.Core/Stryker.Core/TestRunners/VsTest/VsTestContextInformation.cs

```diff
@@ -148,6 +148,7 @@ private ConsoleParameters DetermineConsoleParameters(string runnerId)
                 return determineConsoleParameters;
             }
 
+            determineConsoleParameters.TraceLevel = Options.DevMode ? TraceLevel.Verbose : TraceLevel.Info;
```

Comment:
Does vstest expose user information in Verbose mode that is not exposed in Info mode? Or what is the reason for using devmode here?

Reply:
not sure what you mean by this question...
Verbose expose more information about what is happening inside VsTest, and this was needed for me to understand the problem with it (VsTest).
As I see things, we know have `log-to-file` for when we need a user to share a log and `dev mode` for when a Stryker contributor needs to understand what happens in details.

Reply:
I understand the change for stryker's own logs but this log level is only for vstest, I'm asking what the purpose is for a difference between trace and debug in vstest logs specifically.

Reply:
for vstest logs, there is no debug level. This is information or verbose

Reply:
verbose provides information regarding the exchange between the VsTest server and the test host instances